packages/markdown-medic/src/rules/no-dead-links.ts

```
@@ -0,0 +1,56 @@
+import type { CheckResult } from '../check.js';
+import { existsSync } from 'fs';
+import { resolve, dirname } from 'path';
+
+export async function checkNoDeadLinks(
+  content: string,
+  filePath: string
+): Promise<CheckResult[]> {
```

ESLint

[error] Array type using 'CheckResult[]' is forbidden. Use 'Array<CheckResult>' instead.

(@typescript-eslint/array-type)